jobs/src/main/scala/dcos/metronome/jobrun/impl/JobRunExecutorActor.scala

```diff
@@ -82,13 +89,26 @@ class JobRunExecutorActor(
     context.become(starting)
   }
 
+  def scheduleTimeout(): Unit = {
+    import scala.concurrent.duration._
+
+    jobRun.startingDeadline.foreach { deadline =>
+      val now = clock.now()
+      val from = run.createdAt.plus(deadline.toMillis)
+      val timeout = Seconds.secondsBetween(now, from).getSeconds.seconds
+      log.info(s"Timeout scheduled for ${jobRun.id} at $timeout")
```

Comment:
I much prefer more context in the log here.  we need to know what timeout... and we need to know the configuration and what was calculated.
`s"StartingDeadline timeout of $timeout created for JobRun ${jobRun.id} with createdAt of ${jobRun.createdAt} and startingDeadline of ${deadline} "`